src/generate_databases/pml_set_local_dampingcoeff.f90

```diff
@@ -1362,6 +1379,189 @@ subroutine pml_set_local_dampingcoeff(myrank,xstore,ystore,zstore)
     enddo
   enddo !ispec_CPML
 
+! for robust parameter separation of PML damping parameter
+  minimum_for_dx = minval(d_store_x)
+  minimum_for_dy = minval(d_store_y)
+  minimum_for_dz = minval(d_store_z)
+  maximum_for_dx = maxval(d_store_x)
+  maximum_for_dy = maxval(d_store_x)
```

Comment:
Dear Daniel,

Thank you so much.
I have retested the code and correct the error according to your instruction.

Thanks again,
Best regards,
Zhinan






At 2015-11-16 14:19:38, "daniel peter" <notifications@github.com> wrote:


In src/generate_databases/pml_set_local_dampingcoeff.f90:

> @@ -1362,6 +1379,189 @@ subroutine pml_set_local_dampingcoeff(myrank,xstore,ystore,zstore)
>      enddo
>    enddo !ispec_CPML
>  
> +! for robust parameter separation of PML damping parameter
> +  minimum_for_dx = minval(d_store_x)
> +  minimum_for_dy = minval(d_store_y)
> +  minimum_for_dz = minval(d_store_z)
> +  maximum_for_dx = maxval(d_store_x)
> +  maximum_for_dy = maxval(d_store_x)


should be: maximum_for_dy = maxval(d_store_y)

—
Reply to this email directly or view it on GitHub.